### PR TITLE
QOL stuff

### PR DESCRIPTION
**Pull request by 6SpadesLuke**
# Description

Adds the ability to craft healing poultice so you don't have to only rely on antidote to cure poisoning and also acts like a slightly better healing powder.

---

# TODO

- [x] Completed Task
It's done as far as I can tell
---

### CODE REVIEW

**Comment by 6SpadesLuke** [comment]
I THINK I did it right this time

File: Resources/Prototypes/_Nuclear14/Loadouts/items.yml [hunk 1]
```diff
@@ -193,6 +200,13 @@
   cost: 1
   items:
     - Matchbox
+
+- type: loadout
+  id: N14FoodTinC2
+  category:Items
```
Comment (Peptide90) [inline Resources/Prototypes/_Nuclear14/Loadouts/items.yml:206]: Space missing

File: Resources/Prototypes/_Nuclear14/Loadouts/items.yml [hunk 2]
```diff
@@ -193,6 +200,13 @@
...
+  cost:2
```
Comment (Peptide90) [inline Resources/Prototypes/_Nuclear14/Loadouts/items.yml:207]: Same here

File: Resources/Prototypes/_Nuclear14/Recipes/Lathes/misc.yml [hunk 3]
```diff
@@ -83,4 +83,25 @@
   result: Matchbox
   completetime: 1
   materials:
-    Wood: 100
\ No newline at end of file
+    Wood: 100
+
+- type: latheRecipe
```
Comment (Peptide90) [inline Resources/Prototypes/_Nuclear14/Recipes/Lathes/misc.yml:88]: These recipes probably need added to the entity you make them on. E.g forge or workbench

**Review by Peptide90 — CHANGES_REQUESTED** [review]

**Comment by 6SpadesLuke** [comment]
okay pep waitin on this now

File: Resources/Prototypes/_Nuclear14/Recipes/Crafting/medical.yml [hunk 4]
```diff
@@ -58,6 +58,36 @@
...
+  graph:
+    - node: start
+      edges:
+        - to: healingpoultice
+          steps:
+            - material: BrocFlower
+            - material: XanderRoot # TODO: Same as healing powder for XanderRoot
+            - material: WildBananaYucca
+
+            -node: healingpoultice
```
Comment (Peptide90) [inline Resources/Prototypes/_Nuclear14/Recipes/Crafting/medical.yml:88]: This has incorrect indentation, bring it left in line with the start node. Compare it to other construction graphs too to make sure everything is correct.

**Review by Peptide90 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_Nuclear14/Recipes/Crafting/medical.yml (hunk 4 above)
Comment (6SpadesLuke) [reply Resources/Prototypes/_Nuclear14/Recipes/Crafting/medical.yml]: fixed this

File: Resources/Prototypes/_Nuclear14/Loadouts/items.yml [hunk 5]
```diff
@@ -180,6 +180,13 @@
   items:
     - Lighter
 
+- tpye: loadout
```
Comment (Peptide90) [inline Resources/Prototypes/_Nuclear14/Loadouts/items.yml:183]: Spelling

**Comment by VlaDOS1408** [comment]
YAML linter speak of error in loadouts. Check and fix please. Details in details of checking.

**Comment by 6SpadesLuke** [comment]
I'm so fucking cooked rn...

**Comment by 6SpadesLuke** [comment]
alright final decision : 
I either cut the PR into multiple things (easier)
OR
I stop being a lazy fuck and lock in (difficult)

**Comment by 6SpadesLuke** [comment]
I'm gonna close this since most of its outdated

**Pull request closed without merging**